docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java

```diff
@@ -0,0 +1,146 @@
+package org.arquillian.cube.docker.drone;
+
+import org.arquillian.cube.docker.drone.util.SeleniumVersionUtil;
+import org.arquillian.cube.docker.drone.util.VolumeUtils;
+import org.arquillian.cube.docker.impl.client.config.Await;
+import org.arquillian.cube.docker.impl.client.config.CubeContainer;
+import org.arquillian.cube.docker.impl.client.config.Image;
+import org.arquillian.cube.docker.impl.client.config.Link;
+import org.arquillian.cube.docker.impl.client.config.PortBinding;
+
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class SeleniumContainers {
+
+    private static final String CHROME_IMAGE = "selenium/standalone-chrome-debug:%s";
+    private static final String FIREFOX_IMAGE = "selenium/standalone-firefox-debug:%s";
+    private static final String VNC_IMAGE = "richnorth/vnc-recorder:latest";
```

Comment:
For now only these two are provided. I would say to wait until someone requests it, since I am not sure that someone wants to use a non official browser docker image. I think that the common scenario is this one.